common/environments/python-sdk-v2/asset.yaml

```diff
@@ -0,0 +1,5 @@
+name: python-sdk-v2
+version: 0.0.1
```

Comment:
Should we use auto-versioning here? (`version: auto`)

Reply:
Added comment below